### ISSUE TITLE
DeprecationWarning: loaderUtils.parseQuery()  when running npm start

### ISSUE THREAD

**Issue by oojr** [mozilla-neutrino/react-starter#5]
```bash 
DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.
```

**Comment by eliperelman** [comment]
This is fixed in the latest version of the preset. We just need to upgrade the Neutrino version here.

**Comment by eliperelman** [comment]
In the meantime, locally you can run:

```bash
❯ yarn upgrade neutrino neutrino-preset-react
```

**Comment by helfi92** [comment]
Fixed with #6
